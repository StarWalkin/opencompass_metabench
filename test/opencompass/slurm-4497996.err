  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s] 33%|███▎      | 1/3 [2:40:33<5:21:06, 9633.06s/it]                                                    33%|███▎      | 1/3 [2:40:34<5:21:06, 9633.06s/it] 67%|██████▋   | 2/3 [5:45:20<2:54:48, 10488.42s/it]                                                     67%|██████▋   | 2/3 [5:45:20<2:54:48, 10488.42s/it]100%|██████████| 3/3 [9:53:55<00:00, 12509.73s/it]  100%|██████████| 3/3 [9:53:55<00:00, 11878.58s/it]
  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4497996 ON gpua010 CANCELLED AT 2024-08-17T15:35:35 DUE TO TIME LIMIT ***
