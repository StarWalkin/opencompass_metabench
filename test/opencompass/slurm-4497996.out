/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/17 05:40:21 - OpenCompass - INFO - Loading math_gen: configs/datasets/math/math_gen.py
08/17 05:40:21 - OpenCompass - INFO - Loading hf_llama3_8b_instruct: configs/models/hf_llama/hf_llama3_8b_instruct.py
08/17 05:40:21 - OpenCompass - INFO - Loading hf_qwen2_7b_instruct: configs/models/qwen/hf_qwen2_7b_instruct.py
08/17 05:40:21 - OpenCompass - INFO - Loading hf_yi_1_5_9b_chat: configs/models/yi/hf_yi_1_5_9b_chat.py
08/17 05:40:21 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 05:40:21 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_054021
08/17 05:40:21 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 05:40:22 - OpenCompass - INFO - Partitioned into 3 tasks.
launch OpenICLInfer[llama-3-8b-instruct-hf/math] on GPU 0
launch OpenICLInfer[qwen2-7b-instruct-hf/math] on GPU 0
launch OpenICLInfer[yi-1.5-9b-chat-hf/math] on GPU 0
08/17 15:34:18 - OpenCompass - INFO - Partitioned into 3 tasks.
